### ISSUE TITLE
Error: F1ScoreMulti not evaluated

### ISSUE THREAD

**Issue by samiit** [fastai/fastai2#422]
When I tried to create a multi-label classification model like:

```
learn = cnn_learner(dls, resnet34, pretrained=True, metrics=[accuracy_multi, F1ScoreMulti])

class BCEWithLogitsLossFlat(BaseLoss):
    "Same as `nn.CrossEntropyLoss`, but flattens input and target."
    def __init__(self, *args, axis=-1, floatify=True, thresh=0.35, **kwargs):
        super().__init__(nn.BCEWithLogitsLoss, *args, axis=axis, floatify=floatify, is_2d=False, **kwargs)
        self.thresh = thresh

    def decodes(self, x):    return x>self.thresh
    def activation(self, x): return torch.sigmoid(x)

learn.loss_func = BCEWithLogitsLossFlat()
learn.fine_tune(4)
```

It gives and error of operations. Using only accuracy_multi it gives fair answer. I would like to use the F1ScoreMulti metrics.

Kindly suggest if it is to be fixed in fastai2 or am I doing something wrong?

Sam

**Comment by davanstrien** [comment]
@samiit you need to first instantiate ```F1ScoreMulti``` before you can use it. You can either do something like 

``` 
F1 = F1ScoreMulti()
learn = cnn_learner(dls, resnet34, pretrained=True, metrics=[accuracy_multi, F1ScoreMulti])
```
or
```
learn = cnn_learner(dls, resnet34, pretrained=True, metrics=[accuracy_multi, F1ScoreMulti()])
```

See thread on the forums for more detail [https://forums.fast.ai/t/problem-with-f1scoremulti-metric/63721]()

**Comment by samiit** [comment]
Thank you (for the answer and care for my beginner's question). I missed the fact that I must instantiate an instance of the class.

Sam
